lib/components/Icon/Icon.module.scss

```diff
@@ -11,6 +11,7 @@
     display: inline-flex;
     align-items: center;
     justify-content: center;
+    padding: 1px;

```

Comment:
why do you need 1px?

Reply:
I just made the icons look better, since before we were not actually constraining the size correctly there used to be spaces that this padding now provides... I can remove it tho.